dom.bs

```diff
@@ -1217,7 +1217,32 @@ for discussion).
    <a>invoke</a> algorithm.
 
    <li><p>If <var>event</var>'s {{Event/type}} attribute value is not <var>listener</var>'s
-   <b>type</b>, terminate these substeps (and run them for the next <a>event listener</a>).
+   <b>type</b>, check for a legacy type attribute match by running these substeps:
+
+   <ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationend":
+    <ol>
+     <li><p>Let <var>original event type</var> be "animationend".
+     <li><p>Change the <var>event</var>'s {{Event/type}} attribute value to "webkitanimationend".
+    </ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationiteration":
+    <ol>
+     <li><p>Let <var>original event type</var> be "animationiteration".
+     <li><p>Change the <var>event</var>'s {{Event/type}} attribute value to "webkitanimationiteration".
+    </ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "animationstart":
+    <ol>
+     <li><p>Let <var>original event type</var> be "animationstart".
+     <li><p>Change the <var>event</var>'s {{Event/type}} attribute value to "webkitanimationstart".
+    </ol>
+    <li><p>If <var>event</var>'s {{Event/type}} attribute value is "transitionend":
+    <ol>
+     <li><p>Let <var>original event type</var> be "transitionend".
+     <li><p>Change the <var>event</var>'s {{Event/type}} attribute value to "webkittransitionend".
```

Comment:
Please put newlines between the `<li>`s. (Though not before the first and after the last `<li>` within an `<ol>`.)